src/ci/docker/run.sh

```diff
@@ -93,8 +93,7 @@ if [ -f "$docker_dir/$image/Dockerfile" ]; then
     docker --version
 
     REGISTRY=ghcr.io
-    # PR CI runs on rust-lang, but we want to use the cache from rust-lang-ci
-    REGISTRY_USERNAME=rust-lang-ci
+    REGISTRY_USERNAME=${GITHUB_REPOSITORY_OWNER}
```

Comment:
This means that it's no longer to use the docker cache in local builds.

Reply:
Damn, I forgot to fix this. I will send a PR later today, or tomorrow at the latest.